diff -Nru uclient-0.15.1.orig/Makefile.am uclient-0.15.1/Makefile.am
--- uclient-0.15.1.orig/Makefile.am	Fri Oct 18 18:31:22 2002
+++ uclient-0.15.1/Makefile.am	Thu Apr 24 14:17:07 2003
@@ -65,8 +65,8 @@
 	chmod 777 $(DESTDIR)$(datadir)/uclient/help
 	cp $(top_srcdir)/docs/help.html $(DESTDIR)$(datadir)/uclient/help
 
-	mkdir -p $(DESTDIR)$(localstatedir)/uclient
-	chmod 777 $(DESTDIR)$(localstatedir)/uclient
+	mkdir -p $(DESTDIR)$(localstatedir)/games/uclient
+	chmod 777 $(DESTDIR)$(localstatedir)/games/uclient
 
 	@echo 
 	@echo "UClient is now installed in "$(prefix)
diff -Nru uclient-0.15.1.orig/clear_cache.sh uclient-0.15.1/clear_cache.sh
--- uclient-0.15.1.orig/clear_cache.sh	Tue Apr 10 03:51:10 2001
+++ uclient-0.15.1/clear_cache.sh	Thu Apr 24 14:11:50 2003
@@ -1,3 +1,3 @@
 #!/bin/sh
 find . -name "*.raw" -exec rm -f \{\} \; 
-rm -f ../var/uclient/cache_index.xml
+rm -f /var/games/uclient/cache_index.xml
diff -Nru uclient-0.15.1.orig/media/sprite_cache.cc uclient-0.15.1/media/sprite_cache.cc
--- uclient-0.15.1.orig/media/sprite_cache.cc	Fri Oct 18 18:19:10 2002
+++ uclient-0.15.1/media/sprite_cache.cc	Thu Apr 24 14:12:57 2003
@@ -63,7 +63,7 @@
   varconf::Config::inst()->
     sigv.connect(SigC::slot(this, &SpriteCache::configCallback));  
 
-  std::string indexname = std::string(VARDIR)+"/uclient/cache_index.xml";
+  std::string indexname = std::string(VARDIR)+"/games/uclient/cache_index.xml";
   
   DII::Object index;
   DII::Object::MapType entry;
@@ -180,7 +180,7 @@
 	  itr++;
 	}
       
-      std::string indexname = std::string(VARDIR)+"/uclient/cache_index.xml";
+      std::string indexname = std::string(VARDIR)+"/games/uclient/cache_index.xml";
       
       debugN(8,cerr << " dumping  "<<index.size()<<" index entry to "<<indexname;);
       if(DII::Serialize::write(index, indexname.c_str()))
